### DISCUSSION TITLE
Request for Time series modules

### DISCUSSION THREAD

**Discussion by sathishkumar999** [pycaret/pycaret#1336] [Ideas]
Hi Pycaret Team...
I've been using pycaret modules since 1 month....It makes most of the models automative.
All data science modules are available, except  for time series modules.
could you guys add Time series modules in next version such as "Arima and Holt-Winters".
where most of the retail companies are using for forecasting the monthly, quarterly and yearly product sales.

**Comment by pycaret** [comment]
@sathishkumar369 Thanks for your feedback. This is surely in the list.

Keeping this issue open and tagging this as an enhancement.

**Comment by pycaret** [comment]
@ngupta23 FYI.

**Comment by ngupta23** [comment]
Hi @sathishkumar999 and @balins , This is currently in work. A `beta` version should be ready shortly. Would you be interested in being one of the testers for this module?

**Comment by balins** [comment]
> Hi @sathishkumar999 and @balins , This is currently in work. A `beta` version should be ready shortly. Would you be interested in being one of the testers for this module?

Hi @ngupta23, sorry, but I'm no longer into time series analysis.

**Comment by IlicicAxel** [comment]
Hi @ngupta23 @pycaret , I started using Pycaret hoping classical models would be available. Is there any chance I can be a tester for this module? Thanks in advance!

**Reply by ngupta23** [reply]
Absolutely. Are you on the slack channel? If not, you can join there and we will make an announcement shortly (hopefully by end of month) on the alpha release. https://join.slack.com/t/pycaret/shared_invite/zt-row9phbm-BoJdEVPYnGf7_NxNBP307w
